administrator/install/sql/updates/mysql/4.0.14.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE `#__jcomments` CHANGE `date` `date` DATETIME NOT NULL, CHANGE `checked_out` `checked_out` INT(11) UNSIGNED NULL, CHANGE `checked_out_time` `checked_out_time` DATETIME NULL;
+ALTER TABLE `#__jcomments_blacklist` CHANGE `created` `created` DATETIME NOT NULL, CHANGE `expire` `expire` DATETIME NOT NULL, CHANGE `checked_out` `checked_out` INT(11) UNSIGNED NULL, CHANGE `checked_out_time` `checked_out_time` DATETIME NULL;
+ALTER TABLE `#__jcomments_custom_bbcodes` CHANGE `checked_out` `checked_out` INT(11) UNSIGNED NULL, CHANGE `checked_out_time` `checked_out_time` DATETIME NULL;
+ALTER TABLE `#__jcomments_objects` CHANGE `created` `created` DATETIME NOT NULL;
+ALTER TABLE `#__jcomments_objects` CHANGE `modified` `modified` DATETIME NOT NULL;
```

Comment:
This column doesn't exist in install.sql and will throw an error on upgrade from 4.0.12:
Warning
JInstaller::Install: Error SQL Unknown column 'created' in '#__jcomments_objects'

Also, if the table wasn't empty it will need default values. Better to set the default to:

ALTER TABLE `#__jcomments_objects` CHANGE `created` `created` DATETIME NOT NULL CURRENT_TIMESTAMP;
ALTER TABLE `#__jcomments_objects` CHANGE `modified` `modified` DATETIME NOT NULL '1001-01-01 00:00:00';